manywheel/build_libtorch.sh

```diff
@@ -155,16 +155,14 @@ fi
     # Keep debug symbols on debug lib
     strip --only-keep-debug libtorch/lib/libtorch_cpu.so.dbg
 
+    # Remove debug info from release lib
+    strip --strip-debug libtorch/lib/libtorch_cpu.so
+
     # Add a debug link to the release lib to the debug lib (debuggers will then
     # search for symbols in a file called libtorch_cpu.so.dbg in some
     # predetermined locations) and embed a CRC32 of the debug library into the .so
     cd libtorch/lib
 
-    # Clean out debug symbols from all the libs
```

Comment:
These are removed since we only need to strip the libraries built with debug info (before it was everything, now it's just `libtorch_cpu.so`, which gets stripped above).

[Binary sizes look OK](https://app.circleci.com/pipelines/github/pytorch/pytorch/312690/workflows/7f18f397-2150-4447-9b3d-d9e4cbd310e0/jobs/12976773):

```
total 1.1G
drwxr-xr-x 2 root root  183 Apr 29 01:21 .
drwxr-xr-x 1 root root  215 Apr 29 01:19 ..
-rw-r--r-- 1 root root 662M Apr 29 01:21 debug-libtorch-shared-with-deps-1.9.0.dev20210429+cpu-e4cc6f8d.zip
-rw-r--r-- 1 root root 183M Apr 29 01:21 libtorch-shared-with-deps-1.9.0.dev20210429+cpu.zip
-rw-r--r-- 1 root root 183M Apr 29 01:21 libtorch-shared-with-deps-latest.zip
```